src/main/java/Dooke.java

```diff
@@ -7,16 +7,20 @@
  * @version 0.1
  */
 public class Dooke {
+    /**
+     * The main method.
+     * @param args
+     */
     public static void main(String[] args) {
         String logo =
-                "8888b.   dP\"Yb   dP\"Yb  88  dP 888888\n" +
-                 "8I  Yb dP   Yb dP   Yb 88odP  88__\n"+
-                 "8I  dY Yb   dP Yb   dP 88\"Yb  88\"\"\n"+
-                "8888Y\"   YbodP   YbodP  88  Yb 888888";
+                "8888b.   dP\"Yb   dP\"Yb  88  dP 888888\n"
+                 + "8I  Yb dP   Yb dP   Yb 88odP  88__\n"
+                 + "8I  dY Yb   dP Yb   dP 88\"Yb  88\"\"\n"
+                + "8888Y\"   YbodP   YbodP  88  Yb 888888";
 
         System.out.println("Hello from\n" + logo);
 
-        IOManager ioManager = IOManager.getInstance();
+        IoManager ioManager = IoManager.getInstance();
```

Comment:
Not sure where would be a good place to mention this but, while I do think that the naming convention you follow is great, but I'm think it could be hard to understand for some. At the same time, I do also understand that the code will most likely be viewed by people who can understand it in the first place. (I'm assuming people who use github are people who are able to read and understand codes to a considerable extent.) 💭